jobs/JREGIONAL_RUN_MET_GENENSPROD_OR_ENSEMBLESTAT

```diff
@@ -52,18 +51,40 @@ print_info_msg "
 Entering script:  \"${scrfunc_fn}\"
 In directory:     \"${scrfunc_dir}\"
 
-This is the J-job script for the task that runs METplus for ensemble-stat
-on gridded fields by initialization time for all forecast hours.
+This is the J-job script for the task that runs either METplus's 
+gen_ens_prod tool or its ensemble_stat tool for ensemble verification.
 ========================================================================"
 #
 #-----------------------------------------------------------------------
 #
+# Set the name of the MET/METplus tool this task will call.  (Note: "sc"
+# is for "snake case", i.e. using underscores as separators, and "pc" is
+# for "Pascal case", i.e. no separators but first letter of each word
+# capitalized.)
+#
+#-----------------------------------------------------------------------
+#
+valid_vals_MET_TOOL=( "GENENSPROD" "ENSEMBLESTAT" )
+check_var_valid_value "MET_TOOL" "valid_vals_MET_TOOL"
+
+if [ "${MET_TOOL}" = "GENENSPROD" ]; then
```

Comment:
Yeah, it's left over from the time when some things were named "grid_stat_..." and others used "GridStat_...".  Let me take a look and see if I can simplify.

Reply:
@mkavulich I cleaned up the MET/METplus naming code, hopefully to your approval.  I removed the code in the J-jobs, and now the ex-scripts don't need arguments and get the variable `MET_TOOL` directly from the environment.  I added a function (`get_met_metplus_tool_name.sh`) that sets the names of the MET and METplus tools.  I want to keep both names still because they are both currently used; maybe we'll simplify the ex-scripts later and use only one of the names to refer to the tool, but let's leave that for another PR.  Also, I know we want to do all python scripts, but I haven't gotten a chance to do that yet (will do it in a future PR if someone else doesn't get to it before).  Let me know what you think.

Also, I reran the WE2E tests `MET_verification_only_vx` and `MET_ensemble_verification_only_vx`, and the both passed.